cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -219,6 +219,9 @@ spec:
           value: "true"
         - name: ROOK_CSI_ENABLE_GRPC_METRICS
           value: "true"
+        # Enable deployment of snapshotter container in ceph-csi provisioner
+        - name: CSI_ENABLE_SNAPSHOTTER
+          value: "true"
```

Comment:
I guess I just realized that but shouldn't this be false for openshift?

Reply:
This should be set to false by ocs-operator

Reply:
Ok, sounds good then.